components/backup-stream/src/errors.rs

```diff
@@ -18,16 +19,19 @@ use crate::{endpoint::Task, metrics};
 
 #[derive(ThisError, Debug)]
 pub enum Error {
-    #[error("gRPC meet error {0}")]
-    Grpc(#[from] GrpcError),
-    #[error("Protobuf meet error {0}")]
-    Protobuf(#[from] ProtobufError),
     #[error("No such task {task_name:?}")]
     NoSuchTask { task_name: String },
     #[error("Observe have already canceled for region {0} (version = {1:?})")]
     ObserveCanceled(u64, RegionEpoch),
     #[error("Malformed metadata {0}")]
     MalformedMetadata(String),
+    #[error("Out of quota for region {region_id}")]
+    OutOfQuota { region_id: u64 },
```

Comment:
Why we need the region id?

Reply:
Because we have aborted the initial scanning of this region, then we must deregister the region and reinitialize it then.